xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/DeletePageConfirmationPage.java

```diff
@@ -55,7 +56,16 @@ public void selectOptionSkipRecycleBin()
     }
 
     /**
-     * Confirm the deletion of the page
+     * @return {@code true} if the form proposing to skip the recycle bin is displayed, {@code false} otherwise
+     */
+    public boolean isRecycleBinOptionsDisplayed()
+    {
+        return this.getDriver().findElementsWithoutWaiting(By.cssSelector("input[name='shouldSkipRecycleBin']")).size()
```

Comment:
why are you checking the radio buttons instead of the full control? The number 2 feels magical.